### ISSUE TITLE
Using SDA parallel to the Steam Authenticator App

### ISSUE THREAD

**Issue by OLLI-S** [Jessecar96/SteamDesktopAuthenticator#612]
I have already set up 2FA in Steam for many months and I am also using the Steam Authenticator App on Android. Now I want to extract the Authenticator Key so my password manager KeePassXC can generate the Steam 2FA codes for me.

When I use the Steam Desktop Authenticator (SDA) and log in, I get the following message:

![image](https://user-images.githubusercontent.com/8037790/73487975-4fc37e00-43a8-11ea-8f41-6128f6a4bfc5.png)

I understand that this message is normal, because I am already using an Authenticator App.
And I also understand that I have to remove the current Authenticator and set up the SDA as Authenticator (what makes my current Authenticator App no longer working).

Is there really no way to let the SDA generate the same code that I have on my phone in the Steam App (my phone is not rooted)?

**Reason:** 
I often visit two friends that also play games on Steam. Sometimes I log in at their computer with my steam account to show them some games of my library. When I leave my friends, I log out and they log in using their credentials. With the Steam Authenticator App this is possible (I have it with me).
But in 99% if all cases I log in at home and here I would like to add the Steam Authenticator Key from the SDA to KeePassXC.

**Comment by ChargedByte** [comment]
Steam stores authenticator data in a location that only be accessed by root or the app itself; this is why it is not possible to import accounts from non-rooted devices.

**Comment by Jessecar96** [comment]
It is not possible to extract the secret keys that the Steam Android app uses without rooting your phone. Even if rooted, SDA does not currently have a way to import these keys manually into the program.

**Comment by OLLI-S** [comment]
I just need the secret key so I can add it to KeePassXC.
Is there any tool that can access the file without root?

**Comment by ChargedByte** [comment]
I don't think so, I know some tools can see the files, but it is not possible to open them.

**Comment by IncPlusPlus** [comment]
https://github.com/steamguard-totp/steamguard-shared-secret

**Comment by Teknomancer** [comment]
> I just need the secret key so I can add it to KeePassXC. Is there any tool that can access the file without root?

This looks easier for Steam's 2FA secret key extraction:
https://www.reddit.com/r/Bitwarden/comments/iy4jh4/use_winauth_to_get_steam_and_battlenet_secrets/

That apparently allows extracting the secret key but I didn't manage to get it working yet because it requires a phone number to be associated with Steam on top of Steam Guard being enabled.

**Comment by Hrxn** [comment]
Sure? WinAuth has been discontinued..

**Comment by Teknomancer** [comment]
> Sure? WinAuth has been discontinued..

Sure, but it still works and is able to extract the secret so I don't see why it's a problem.
